code/lib/cli/src/js-package-manager/PNPMProxy.ts

```diff
@@ -34,8 +34,11 @@ export class PNPMProxy extends JsPackageManager {
     return pathExistsSync(pnpmWorkspaceYaml);
   }
 
-  initPackageJson() {
-    return this.executeCommand('pnpm', ['init', '-y']);
+  async initPackageJson() {
+    await this.executeCommand({
```

Comment:
Did you, on purpose, not return `this.executeCommand(...) anymore?

Reply:
Good catch!

Reply:
Yup, there was no reason to return anything!